Framework/API/inc/MantidAPI/AlgorithmHistory.h

```diff
@@ -93,6 +80,8 @@ class MANTID_API_DLL AlgorithmHistory {
   }
   /// get the execution count
   const std::size_t &execCount() const { return m_execCount; }
+  /// get the uuid
+  const std::string uuid() const { return m_uuid; }
```

Comment:
Why force the return to be `const`? Or is `&` missing?

Reply:
& is missing